### DISCUSSION TITLE
Is TE3 Business license sufficient for Fabric workspace XMLA?

### DISCUSSION THREAD

**Discussion by kaidobor** [TabularEditor/TabularEditor3#1373] [🙋♀️TE3 Q&A] [answered]
I am trying to choose a license (Business or Enterprise) for handling semantic models in Fabric F2 workspace. I am able to connect to XMLA endpoint with TE2. Now TE3 license choice is confusing to me. 

In pricing comparison table it is shown that TE Business edition allows "Power BI Premium Per User", but does not specify allowing "Fabric Capacity (F SKUs)". On the other hand it is elsewhere shown that XMLA access exists. So can I access my F2 workspace XMLA with Business edition? Or is TE3 Business edition simply blocking XMLA accessing the workspace with Fabric F2 license to push to using Enterprise license?

Or alternatively, does this extra "Fabric Capacity (F SKUs)" feature in Enterprise edition unlock some Fabric specific features? In such case, would be great to know, what features are then unlocked.

Thanks in advance for info so I can make a choice.

**Comment by otykier** [comment, marked as answer]
Hi @kaidobor 
Thanks for reaching out.

In the context of Power BI / Fabric:

- TE3 Business Edition **only** allows XMLA endpoint access to Power BI workspaces that are under the Premium-Per-User license
  - In addition, some Fabric-specific features (Direct Lake, etc.), will not work with TE3 Business Edition (i.e. trying to load a Model.bim file which uses Direct Lake mode)
- TE3 Enterprise Edition has no limitations in this regard, and allows full access to all XMLA endpoints (same as TE2)

Hope this helps.
